### PR TITLE
Solar Tower and Steam Squasher Quests

### PR DESCRIPTION
**Pull request by Steelux8**
This PR adds two quests, one for each of the GT++ multis in the title. Also changes to the new algae farm quest. More quests for other multis will be coming soon, but it's late and I wanted to push current changes.

(I left edit mode on again. I'm not sure where it's saved in the files after the refactoring, if it even is saved in a file that gets pushed in these PRs.)

### CODE REVIEW

**Comment by chochem** [comment]
edit more isnt saved anymore. nice qol from a few months ago

File: config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AlgaePond-Ds_ok-hRS2ODJNs3LUP9kw==.json [hunk 1]
```diff
@@ -69,27 +69,17 @@
           "OreDict:8": ""
         },
         "1:10": {
-          "id:8": "miscutils:item.BasicAgrichemItem",
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AlgaePond-Ds_ok-hRS2ODJNs3LUP9kw==.json:72]: why did you remove them? that makes no sense. I even explained that to you. please revert that

**Review by chochem — DISMISSED** [review]
see line comment

File: config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AlgaePond-Ds_ok-hRS2ODJNs3LUP9kw==.json (hunk 1 above)
Comment (Steelux8) [reply config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AlgaePond-Ds_ok-hRS2ODJNs3LUP9kw==.json:72]: Because they're unobtainable items at the first tiers of the Algae Farm, which are what the quest is for. In general, quests don't give unobtainable items as rewards with few exceptions, so I automatically removed all algae but the first one. Didn't think about it.

Comment (chochem) [reply config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AlgaePond-Ds_ok-hRS2ODJNs3LUP9kw==.json:72]: i dont the algae by themselves are particularly useful to skip anything. you can make it just 1 if you think they are.

**Comment by Dream-Master** [comment]
works
image
